pybamm/experiment/experiment.py

```diff
@@ -20,9 +20,9 @@ class Experiment:
     ----------
     operating_conditions : list[str]
         List of strings representing the operating conditions.
-    period : string, optional
+    period : str, optional
         Period (1/frequency) at which to record outputs. Default is 1 minute. Can be
-        overwritten by individual operating conditions.
+        overwritten by individual operating conditions. e.g. 30 minute, 1 hour.
```

Comment:
I don't think the 30 min part adds anything here, if anything it makes it more confusing. 
```suggestion
        overwritten by individual operating conditions.
```

Reply:
Following the suggested change, I have removed it. 
Shall we add '_e.g. in second, minute, hour_' to describe the possible string values in period? 